docs/_sass/_search-bar.scss

```diff
@@ -4,7 +4,10 @@
 
 $color-appBG: $color-green-appBG;
 $color-highlightBG: $color-green-highlightBG;
+$color-highlightBG-hover: $color-green-highlightBG-hover;
```

Comment:
I don't understand these variables. You should just set and declare your color variables in colors.scss, and then you can use them in this file without needing to re-state them.

Reply:
This is the pattern we use on main.scss 

So in this case we will use directly on search-bar related styles?

Reply:
Yeah, I don't know why we need to write the same vars twice. Can't they just live in colors, and then we call them as we need them?

Reply:
I don't know what the initial thought behind it is, but I think the main reason is :
- Let's say we are working on `search-bar.scss` we didn't have to look back to back to `colors.scss`, as reassigning them on the file makes things easier.